be/src/exprs/vectorized/runtime_filter.h

```diff
@@ -336,13 +337,51 @@ class RuntimeBloomFilter final : public JoinRuntimeFilter {
     }
 
     size_t evaluate(Column* input_column, RunningContext* ctx) const override {
-        if (_hash_partition_number != 0) {
+        if (_num_hash_partitions != 0) {
             return t_evaluate<true>(input_column, ctx);
         } else {
             return t_evaluate<false>(input_column, ctx);
         }
     }
 
+    void compute_hash_values_for_multi_part(RunningContext* running_ctx, int8_t join_mode, const Column* input_column,
+                                            std::vector<uint32_t>& hash_values, size_t num_rows) const {
+        typedef void (Column::*HashFuncType)(uint32_t*, uint32_t, uint32_t) const;
+
+        auto compute_hash = [&input_column, &num_rows, &hash_values, this](HashFuncType hash_func,
+                                                                           size_t num_hash_partitions) {
+            hash_values.assign(num_rows, HashUtil::FNV_SEED);
+            (input_column->*hash_func)(hash_values.data(), 0, num_rows);
+            for (size_t i = 0; i < num_rows; i++) {
+                hash_values[i] %= num_hash_partitions;
+            }
+        };
+
+        switch (join_mode) {
+        case TRuntimeFilterBuildJoinMode::BORADCAST: {
+            hash_values.assign(num_rows, 0);
```

Comment:
Why need to assign hash value for boradcast join?

Reply:
keep consistent with the old code. delete it ?